Core/GameInstance.py

```diff
@@ -20,65 +21,19 @@ def __init__(self, window):
 
         # Core game resources
         self.game_resources = GameResources()
+
+        # Managers
+        self.object_manager = ObjectManager(self.game_resources)
         self.projectile_manager = ProjectileManager(self.game_resources)
+        self.light_manager = LightManager(self.game_resources, window)
 
         # Physics engine
         self.physics_engine = setup_physics_engine(self.game_resources)
 
         self.horizontal_key_list = []
         self.verticle_key_list = []
 
-        # create default scene renderer via factory.
-        # This configures the post processing stack and default lighting
-        self.scene_renderer = RendererFactory.create_renderer(window)
-
-        # bind rendering callbacks
-        self.scene_renderer.draw_primary_callback = self.on_draw_scene
-        self.scene_renderer.draw_emissive_callback = self.on_draw_emissive
-        self.scene_renderer.draw_after_post_callback = self.on_draw_after_post
-
-        # Set background color
-        # Based on old arcade.AMAZON color
-        # (59, 122, 87)
-        self.scene_renderer.background_color = (
-            59.0 / 255.0,
-            122.0 / 255.0,
-            87.0 / 255.0,
-            1.0,
-        )
-
-        # dim the ambient lighting to make the player's light more vibrant
-        self.scene_renderer.light_renderer.ambient_light = (0.25, 0.25, 0.25)
-
-        # create light sources
-        self.light_list = []
-
-        self.player_light = self.scene_renderer.light_renderer.create_point_light(
-            (400, 400),  # Position
-            (
-                1.75,
-                1.75,
-                1.75,
-            ),  # Color, 0 = black, 1 = white, 0.5 = grey, order is RGB This can go over 1.0 because of HDR
-            160.0,
-        )  # Radius
-
-        # dict used to determine radius of light based on light_type
-        radius_by_type = {"torch": 70.0, "candle": 40.0}
-
-        for light in self.game_resources.light_list:
-            radius = radius_by_type.get(light.properties["type"])
-            self.light_list.append(
-                self.scene_renderer.light_renderer.create_point_light(
-                    (light.center_x, light.center_y),  # Position
-                    (
-                        1.75,
-                        2.75,
-                        1.75,
-                    ),  # Color, 0 = black, 1 = white, 0.5 = grey, order is RGB This can go over 1.0 because of HDR
-                    radius,
-                )  # Radius
-            )
+        
```

Comment:
Moving this lighting related code into the light manager might make sense, but moving the scene rendered allocation and instance itself into the light manager is probably not the way we want to go